modules/social_features/social_event/config/optional/message.template.user_was_enrolled_to_event.yml

```diff
@@ -0,0 +1,40 @@
+langcode: en
+status: true
+dependencies:
+  module:
+  - activity_logger
+  - social_event_managers
+  enforced:
+    module:
+      - social_event
+third_party_settings:
+  activity_logger:
+    activity_bundle_entities:
+      event_enrollment-event_enrollment: event_enrollment-event_enrollment
+    activity_action: create_entitiy_action
+    activity_context: owner_event_enrollment_activity_context
+    activity_destinations:
+      email: email
+    activity_create_direct: true
+    activity_aggregate: false
+    activity_entity_condition: event_enrollment_standalone_enroll
+    email_subject: 'You were enrolled to an event'
+template: user_was_enrolled_to_event
+label: 'A user was enrolled to an Event'
+description: 'A user was enrolled to an Event'
```

Comment:
Not sure, but maybe `A user has enrolled in an Event`?

Reply:
@rollki changed with suggestions provided by Jessica.